drivers/api/no_os_eeprom.c

```diff
@@ -41,6 +41,12 @@
 #include <stdlib.h>
 #include "no_os_eeprom.h"
 #include "no_os_error.h"
+#include "no_os_mutex.h"
+
+/**
+ * @brief - EEPROM mutex
+*/
+static void *eeprom_mutex;
```

Comment:
i'm not sure i understand the role of this global

Reply:
(same comment for all the other files)

Reply:
This pointer is initialized only once to point to the allocated mutex. Otherwise is just a null pointer.

Reply:
Similar to this, but having only one element:
https://github.com/analogdevicesinc/no-OS/blob/dea741fba32637889047a3eba778565f3e3ecfe3/drivers/api/no_os_i2c.c#L50

Reply:
Will change it to array